src/include/miopen/any_solver.hpp

```diff
@@ -94,7 +95,18 @@ struct AnySolver
     {
         assert(ptr_value != nullptr);
         return ptr_value->FindSolution(ctx, problem, db, invoke_ctx, perf_cfg);
-    };
+    }
+
+    ConvSolution FindSolution(const ExecutionContext& ctx,
```

Comment:
Let's add `#define FIN_ANY_SOLVER_FIND_SOLUTION_COMPAT 1` and wrap old `FindSolution()` with `#if/#endif`.
This code should be removed after the `Fin` is fixed.